api/user.go

```diff
@@ -16,19 +16,22 @@ func (r GetUserRequest) ValidationRules() []validate.ValidationRule {
 }
 
 type User struct {
-	ID            uid.ID   `json:"id"`
-	Created       Time     `json:"created"`
-	Updated       Time     `json:"updated"`
-	LastSeenAt    Time     `json:"lastSeenAt"`
-	Name          string   `json:"name"`
-	ProviderNames []string `json:"providerNames,omitempty"`
+	ID            uid.ID          `json:"id"`
+	Created       Time            `json:"created"`
+	Updated       Time            `json:"updated"`
+	LastSeenAt    Time            `json:"lastSeenAt"`
+	Name          string          `json:"name"`
+	ProviderNames []string        `json:"providerNames,omitempty"`
+	PublicKeys    []UserPublicKey `json:"publicKeys,omitempty"`
+	SSHUsername   string          `json:"sshUsername"`
```

Comment:
Is there a way we can keep this less specific to SSH? Other destinations will have similar limitations in usernames (e.g. example Kubernetes service accounts are restricted to the Kubernetes resource naming schema).

Reply:
Ya, there is discussion about this over in https://github.com/infrahq/infra/pull/3620#pullrequestreview-1179545073

I'd like to find a better name for this field